docs/tutorials/create-on-chain-blog/create-new-site.mdx

```diff
@@ -199,17 +199,33 @@ export async function signTransaction(
 
 This `signTransaction` function streamlines the process of signing transactions. It extracts `signingEntries` from the provided `txSkeleton` and signs the messages within using MetaMask. After signing, it handles the ECDSA recoveryID v parameter and uses `@ckb-lumos/lumos` methods to pack the signature into the Omnilock witness. This process results in the generation of a signed transaction that is ready to send to the Nervos CKB blockchain.
 
-**3. Create Cluster** 
+**3. Share SporeConfig**
```

Comment:
is there an alternative verb for "share" SporeConfig?

Reply:
I don't have a better option in mind, how about "use"? Call for help @ashuralyk @Flouse.

Reply:
What about `Export SporeConfig`?

`export` is a common concept/keyword in both JS/TS and bash.

Reply:
export is better than share imo

Reply:
> "export" reads better than "share" imo

Editied